components/root/root.test.jsx

```diff
@@ -61,34 +66,46 @@ describe('components/Root', () => {
         },
     };
 
+    const origCookies = document.cookie;
+    const origWasLoggedIn = localStorage.getItem('was_logged_in');
+
+    beforeAll(() => {
+        document.cookie = '';
+        localStorage.setItem('was_logged_in', '');
+    });
+
+    afterAll(() => {
+        document.cookie = origCookies;
+        localStorage.setItem('was_logged_in', origWasLoggedIn);
+    });
+
     test('should load config and license on mount and redirect to sign-up page', () => {
         const props = {
             ...baseProps,
             noAccounts: true,
-            actions: {
-                ...baseProps.actions,
-                loadMeAndConfig: jest.fn(async () => [{}, {}, {}]),
-            },
             history: {
                 push: jest.fn(),
             },
         };
 
         const wrapper = shallow(<Root {...props}/>);
 
-        expect(props.actions.loadMeAndConfig).toHaveBeenCalledTimes(1);
-
         wrapper.instance().onConfigLoaded();
         expect(props.history.push).toHaveBeenCalledWith('/signup_user_complete');
         wrapper.unmount();
     });
 
     test('should load user, config, and license on mount and redirect to defaultTeam on success', (done) => {
+        document.cookie = 'MMUSERID=userid';
```

Comment:
Not sure what others see, but Github is showing me a CodeQL warning that looks like:

![false-positive](https://user-images.githubusercontent.com/13738432/159780609-3b7843f5-9091-445b-b6b9-6b4d697b2495.png)

I think its a false positive for two reasons:

1) The explanation seems to treat it as server code, which it isn't. It's client based code mimicking what a server could do.
2) It's test code, not app code.

How do we flag things as false positive here in MM? Any special process?

Reply:
Maybe there's an option under that dismiss dropdown?